scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -94,6 +94,9 @@ make -j 4
 
 cd ..
 mkdir -p upload
+# Patch soljson.js to provide backwards-compatibility with older emscripten versions
+echo ";/* backwards compatibility */ Module['Runtime'] = Module;" >> $BUILD_DIR/libsolc/soljson.js
```

Comment:
This line was originally added in #5831 for 0.5.3, so we need to keep it til 0.6.0.